src/js/main.js

```diff
@@ -1 +1,26 @@
+const taskInput = document.querySelector('#task');
+const taskForm = document.querySelector('#taskForm');
+const taskList = document.querySelector('#taskList');

```

Comment:
use '$' for each const that selects an html element.

example, change 'taskList' into '$taskList'